includes/plugins/google-site-kit/class-googlesitekit-logger.php

```diff
@@ -30,6 +30,12 @@ class GoogleSiteKit_Logger {
 	 * Initialize hooks and filters.
 	 */
 	public static function init() {
+		if (
+			method_exists( 'Newspack_Manager', 'is_connected_to_production_manager' )
+			&& ! \Newspack_Manager::is_connected_to_production_manager()
```

Comment:
```suggestion
			! method_exists( 'Newspack_Manager', 'is_connected_to_production_manager' )
			|| ! \Newspack_Manager::is_connected_to_production_manager()
```

This suggestion is more of a question, shouldn't it also bail if the manager doesn't exist?

Reply:
The intention was to keep it as it was on self-hosted. Someone might need the logging.